src/helpers/git_helpers/branch_configuration_helpers.sh

```diff
@@ -94,89 +123,72 @@ function echo_update_child_branches {
 
 # Makes sure that we know all the parent branches
 # Asks the user if necessary
-# Aborts the script if not all branches become known.
+#
+# branches - space seperated list of branches
```

Comment:
How do you feel about adding this comment to line 129, where the parameter is named? This way there is less redundancy between naming parameters in code, and describing them a little through a comment.